src/devices/ExplorerHat/ExplorerHat.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
@joperezr will probably ask you to remove sln :smile:

Reply:
I forgot to remove that...